driver-sync/src/test/functional/com/mongodb/client/AbstractChangeStreamsTest.java

```diff
@@ -14,14 +14,12 @@
  * limitations under the License.
  */
 
-package com.mongodb.async.client;
+package com.mongodb.client;
```

Comment:
The way git interpreted these changes is unfortunate.  What I actually did was delete the async ChangeStreamsTest and rename the sync one to AbstractChangeStreamsTest.  But git saw that as a rename of the async implementation

Reply:
Interesting fact git is a content tracker and does not track file names in commits. The only way for git log to track it as a delete and a rename would be to do it in multiple commits.